### ISSUE TITLE
Elasticstack 7 not working after install

### ISSUE THREAD

**Issue by stoeps13** [HCL-TECH-SOFTWARE/connections-automation#200]
Hi,
I tried to install Elasticstack 7 today and set the vaiable `setup_elasticstack7=true` in my variables file.

The pods are not starting:
```
kubectl get pods -l release=elasticstack
NAME                        READY   STATUS             RESTARTS   AGE
filebeat-7-8mk9j            0/1     InvalidImageName   0          16h
kibana-7-65994b6c5d-vhb4g   0/1     InvalidImageName   0          16h
logstash-7-0                0/1     InvalidImageName   0          16h
```

`kubectl describe pod -l app=kibana-7`
```
Containers:
  kibana-7:
    Container ID:   
    Image:          map[repository:connections-docker.artifactory.cwp.pnp-hcl.com tag:20210418-230758]:7.6.1
    Image ID:       
    Port:           5601/TCP
```

There is still a helmchart for elasticstack available, but the created pods do not work, because the container images are missing. Even fixing the image name will not help. So I checked the documentation on https://help.hcltechsw.com/connections/v7/admin/install/cp_over_intro.html?hl=kibana and found:

> Elastic Stack is a collection of open-source tools that collect log data and help you visualize those logs in a central location. Specific tools in Elastic Stack include Kibana, Logstash, FileBeat, and Elasticsearch Curator. Starting with version 7, self-build images for Elastic Stack are not provided, as they are no longer needed. Instead, guidance and the documentation on how to set up Elastic Stack, either using the HCL-provided Elasticsearch 7.6.1 or any available Elasticsearch, including AWS native setup.

As there is no additional information available in the documentation, it would be great if this project can deploy a working Elasticstack on top of Connections 7. The default kibana image won't work, as it need to have the Searchguard plugin and Elasticsearch certificates for the connection.
Regards
Christoph

**Comment by sabrina-yee** [comment]
Thanks for reporting the issue @stoeps13.  I understand that a case has been opened for this.  We'll follow up to address this.

**Comment by stoeps13** [comment]
Yes I opened a case too, because documentation mentions that it is possible to install, but not how.

I thought the playbook does the trick, but it just runs the helm chart and the images are missing.

Thanks!

Christoph

**Comment by nbajjuri-hcl** [comment]
@stoeps13 We have a new release of Elasticstack7 helm chart that works as expected. The old one is buggy (do not use it). How would i deliver to you ? You said case was opened.. Do you have case # in reference ? Please share.

**Comment by stoeps13** [comment]
Hi,
I already got the files in the case. Will test today.
Will you update the ansible roles to include the update?
Regards
Christoph

**Comment by nbajjuri-hcl** [comment]
@stoeps13 Since it is not default, it's customers option to change if they want to install.

**Comment by stoeps13** [comment]
When you enable Elastickstack in your variables, the project installs the default helm chart, which ends in errors, because of the missing images.

I would expect a working deployment when I use these scripts.

**Comment by nbajjuri-hcl** [comment]
@stoeps13 Can you replace the default chart with the latest one in the ExtractedFolder ? I will ask my team if we can release another version of Component Pack zip with inclusion of updated ELK7 helm chart.

**Comment by nbajjuri-hcl** [comment]
@stoeps13 Forgot to mention in the case update, Elasticstack7 version is 7.10.1 which requires the Elasticsearch v7.10.1. The Elasticversion 7.10.1 is available on HCL's FlexNet site. The Elasticsearch release is 7.6.1 in the Component Pack zip (7.0.0.2) which was released with CNX v7.0.

**Comment by stoeps13** [comment]
It works with the 7.6.1 ES installation, but I know of the update.
Same here, there should be an option to deploy container updates automatically. (Like the other tools after download from flexnet).

**Comment by stoeps13** [comment]
Nice to have the Kibana 7 config in nginx now, but the helm chart and image is not available on Flexnet for download. Additionally the automation script does not install them (or push the image to registry), the updated Elasticsearch images and charts
![image](https://user-images.githubusercontent.com/1975258/180744053-c42d688e-1db1-4d4c-9448-2f6287b6413c.png)
are not installed at all (and are requirement for the Kibana 7 deployment).

**Comment by sabrina-yee** [comment]
Yes, the delivery of Component Pack updates is not part of the scope of this repo.  It expects they are delivered via the official channel and loaded to the registry prior to running the script.

**Comment by stoeps13** [comment]
Even loaded to the registry would not help, as the tags are different and the helm charts in Cp7.0.0.2 just deploy the pods from May 2021. So even after uploading to the registry, a new image does not get used.

**Comment by sabrina-yee** [comment]
Our Support team is working on addressing this.
